modules/givc/host.nix

```diff
@@ -31,8 +31,11 @@ in
         inherit addr;
         port = "9000";
         services = [
+          "microvm@admin-vm.service"
           "reboot.target"
           "poweroff.target"
+          "sleep.target"
+          "suspend.target"
```

Comment:
These targets are currently not used. We could add a conditional invocation of these targets if the requests originates in gui-vm